### DISCUSSION TITLE
UCUM Internationalization

### DISCUSSION THREAD

**Discussion by smals-jy** [ucum-org/ucum#343] [Q&A]
Hello,

I read https://ucum.org/docs/internationalization and I saw this : 

> UCUM shall invite, promote and help distribute contributions of translations of any UCUM related material, including the text of the specification.
> Since print names are not normative in UCUM, print names may be translated.

I would be interested to see the translation of print name in these main languages : 

- German
- French
- Dutch
- ...

Commercial softwares have their own for them and it would be nice to have open-source versions as well.

Kind regards

**Comment by Medman59** [comment]
I would also like to see French Canadian Print name support. Also available from UCUM and shared with HL7.

**Comment by Medman59** [comment]
Any update of UCUM Internationalization?

**Reply by smals-jy** [reply]
Doesn't seem they made progress on this topic. Personally, it could be a nice asset to improve my libraries that turn [FHIR Dosage](https://build.fhir.org/dosage.html) into human-readable text such as [the Java port](https://github.com/jy95/fds). It might interest you for PrescribeIT @Medman59

**Comment by chgessner** [comment]
The focus of UCUM is on the (international) code system and the (international) representation for print or display, like e. g. described in ISO 80000. These are typically one or two letter symbols, with operators added in some cases (e.g. "km/h" or "IU/mL"), no translations required here. So UCUM is international by definition. We do not curate "long print names" in different languages (e.g. "centimetro", "passo", "pollice" in Italian). Such "translations" may of course be supplemented and shared by interested parties who would require such tables for their use of UCUM.
